### DISCUSSION TITLE
황윤재 - 게임 - 발로란트 유저 데이터 분석

### DISCUSSION THREAD

**Discussion by yoonjae97** [dinnercoding/wanted-pre-onboarding-challenge-data#111] [2023년 11월 프리온보딩 데이터 챌린지]
## 도메인 선택 이유
- 게임을 즐겨하는데 게임 분야의 데이터 분석가로 유저들의 플레이 환경을 개선하는 일을 하면 굉장히 즐겁고 열정적으로 할 수 있을 것 같습니다.
- 그중에서도 즐겨하는 게임의 데이터를 대상으로 분석해보면서 게임 데이터 분석가는 어떤 일을 할 수 있을까에 대해 알아보고자 주제를 선택하게 되었습니다.

## 구체적인 문제 정의
- 인게임 지표 중에서 승률에 영향을 미치는 요소 도출 및 유저의 평균 승률 예측 모델 개발
- 캐릭터(요원)별 지표를 통해 추후 패치에 참고할 수 있도록 지나치게 지표가 좋거나 나쁜 캐릭터들을 도출

## 데이터 업로드
- 데이터세트명 : Valorant Leaderboard Statistics
- 출처 : Kaggle
- 링크 : https://www.kaggle.com/datasets/aliibrahim10/valorant-stats/data

**Comment by dinnercoding** [comment]
@yoonjae97 님! 본인이 좋아하는 분야를 잘 선택해주신 것 같아요. 롤이나 인기 있는 게임의 경우 확실히 유저들의 플레이 경험 만족을 극대화하는 것으로 알고 있는데요! 시즌이 바뀌면서 너무 좋은 캐릭터는 너프를 먹이고 사람들이 잘 안하는 캐릭터는 버프를 먹이는 등 이러한 작업들도 대다수 유저의 플레이 경험을 만족시키기 위함일 것 같고 결국 데이터 분석이 필요하다는 생각이 드네요. 좋은 주제를 선택하신 것 같고 재미있는 시각화 결과가 나오면 좋겠네요 ㅎㅎ
